### DISCUSSION TITLE
Any news Daniel when 4.0.2.4 gets published officially on opencart.com ?

### DISCUSSION THREAD

**Discussion by yamlv** [opencart/opencart#13776] [General]
Hi,
Any news when 4.0.2.4 gets published officially on opencart.com ?

I have at least 4-5 developers who desperately waiting of OCMOD comback, but they dont want to take github version as a base for developing module, they said they will start convert to ocmod 4x only when its official released.

Daniel any news when its will be available?

Thank you.

**Comment by mhcwebdesign** [comment]
If anything, the next release will be 4.1.0.0, not 4.0.2.4, by the looks of it.

**Comment by felipoantonoff** [comment]
I've also been waiting for OCMOD to return for over a year, looking forward to Opencart 4 with OCMOD support, who knows, maybe it will also start to become more popular than Opencart 3 and several Apps/extensions migrated to it.

I hope the next version has support for OCMOD, what you can do is use vQmod and then when OCMOD arrives, switch to it for example, but I also know that it is not ideal.

**Comment by danielkerr** [comment]
i got stuck with a few things which i have solved now but still need to come up with a better solution for the order editor. i think maybe end of the month.

**Reply by felipoantonoff** [reply]
Great, I'm happy to know, I hope that Opencart 4 with the new version becomes increasingly popular and mature.

**Reply by yamlv** [reply]
Thank you Daniel for putting a lot of work and time to opencart, hope you can release it very soon. Ocmod in 4x will 100% boost popularity in opencart 4x and many modules will appear on marketplace too!

**Comment by scubanarc** [comment]
@danielkerr Any thoughts on when 4.1.0.0 is going to drop?

**Reply by felipoantonoff** [reply]
Also waiting for the new Opencart 4 with OCMOD support.
